docs/resources/tutorials/cloud/change_domain_to_ds/change_domain_client.yaml

```
@@ -0,0 +1,22 @@
+version: v3.0
+
+participants:
+
+  - name: ROS_2_Domain_0
+    kind: local
+    domain: 0
+
+  - name: ROS_2_ds_client
+    kind: local-discovery-server
+
+    discovery-server-guid:
+      ros-discovery-server: true
+      id: 2
```

Then, we can use `id: 1` here

```suggestion
      id: 1
```